index.js

```diff
@@ -187,7 +185,15 @@ module.exports = function(robot) {
     if (queue.isEmpty()) {
       res.send('Nobodyz! Like this: []');
     } else {
-      res.send('Here\'s who\'s in the queue: ' + _.pluck(queue.get(), 'name').join(', ') + '.');
+      res.send('Here\'s who\'s in the queue:');
+      for (var x = 0; x < queue.get().length; x++) {
```

Comment:
let's use lodash's for each here. That would save you the temp vars and the multiple calls to queue.get(). Something like:
```
_.forEach(queue.get(), function(item) {....}